platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -66,6 +66,21 @@ TBD:
 - Copy experience
 - Invalidation reason
 
+### Configuration API enhancements
+
+It is now possible to configure a [Configuration](dsl/org.gradle.api.artifacts.Configuration.html) to extend another configuration by supplying a [Provider<Configuration>](dsl/org.gradle.api.provider.Provider.html) as an argument to [extendsFrom](dsl/org.gradle.api.artifacts.Configuration.html#org.gradle.api.artifacts.Configuration:extendsFrom(org.gradle.api.artifacts.Configuration[])).
```

Comment:
💭 TODO: Can this be linked to the proper overload?  Why isn't it appearing in the generated DSL docs?